lib/collections/schemas/sms.js

```diff
@@ -0,0 +1,24 @@
+import { SimpleSchema } from "meteor/aldeed:simple-schema";
+
+export const Sms  = new SimpleSchema({
+  apiKey: {
+    type: String,
+    optional: true
```

Comment:
Are these really all optional except for `shopId`? There are no required fields for this to work? At least `smsProvider`?